app/bundles/PageBundle/Entity/Hit.php

```diff
@@ -558,6 +558,7 @@ public function getUrl()
      */
     public function setUrlTitle($urlTitle)
     {
+        $urlTitle       = mb_strlen($urlTitle) <= 191 ? $urlTitle : substr($urlTitle, 0, 191);
```

Comment:
Let's use the multibyte functions
```suggestion
        $urlTitle       = mb_strlen($urlTitle) <= 191 ? $urlTitle : mb_substr($urlTitle, 0, 191);
```